### PR TITLE
Add ValidationValues mixin

### PR DESCRIPTION
**Pull request by Daenyth**
Similar to the existing DisjunctionValues

### CODE REVIEW

**Comment by coltfred** [comment]
I was just coming here to write this same thing. @bwmcadams, do you have an opinion on the naming of `leftValue` for Validation. I was going to call it `failureValue` since people generally call the error side of Validation `Failure`.

File: src/main/scala/ValidationValues.scala [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  import scala.language.implicitConversions
+
+  /**
+   * Implicit conversion that adds a `value` method to `scalaz.Validation`
+   *
+   * @param validation the `scalaz.Validation` on which to add the `value` method
+   */
+  implicit def convertValidationToValidationable[E, T](validation: Validation[E, T]): Validationable[E, T] = new Validationable(validation)
+
+  // TODO: Fix me as a proper repl session example
```
Comment (coltfred) [inline src/main/scala/ValidationValues.scala]: We should probably take out this TODO.

File: src/main/scala/ValidationValues.scala [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            MonkeyPatch.getStackDepthFun("ValidationValues.scala", "leftValue"))
+        case Failure(left) =>
+          left
+      }
+    }
+  }
+
+}
+
+// TODO: Fix me as a proper repl session example
```
Comment (coltfred) [inline src/main/scala/ValidationValues.scala]: Ditto.

File: src/test/scala/ValidationValuesSpec.scala [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+  describe("leftValue on Validation"){
+    it("should return the value if it's left"){
+      val r = Failure(thisRecord)
+      r.leftValue should === (thisRecord)
+    }
+
+    it("should throw TestFailedException if the validation is right"){
+      val r = Success(thisRecord)
+      val caught = evaluating{
```
Comment (coltfred) [inline src/test/scala/ValidationValuesSpec.scala]: See my PR #4 about replacing the deprecated `evaluating` syntax.

**Comment by curata-falcon** [comment]
@coltfred I went with left/right for symmetry with DisjunctionValues, though I wouldn't object to making it `successValue` and `failureValue`

**Comment by Daenyth** [comment]
I went with left/right for symmetry with DisjunctionValues, but I wouldn't object to them being named `successValue` and `failureValue`

**Comment by coltfred** [comment]
@Daenyth Looks like @bwmcadams decided symmetry was good. :smile: Sorry it took so long for this to get noticed. I don't know that either @bwmcadams or I get notified for issues here. I'll try and change that.

**Comment by Daenyth** [comment]
@coltfred No worries, I'm happy to have it merged. I think the trigger for notifications is the "Watch" button on the top right.

**Pull request merged**